model/state_variables.py

```diff
@@ -81,8 +82,15 @@ class StateVariables:
     """The number of validators in activation queue"""
     average_effective_balance: Gwei = 32 * constants.gwei
     """The validator average effective balance"""
-    number_of_validators: int = number_of_validators
-    """The total number of validators"""
+    number_of_active_validators: int = number_of_active_validators
+    """The total number of active validators"""
+    number_of_awake_validators: int = min(
+        system_parameters.parameters['MAX_VALIDATOR_COUNT'][0],
+        number_of_active_validators
+    )
```

Comment:
Using `min()` with a `None` type will break the code, I suggest the following (it works both for `None` and `float('inf')`):
```suggestion
    number_of_awake_validators: int = min(
        x for x in [
            system_parameters.parameters['MAX_VALIDATOR_COUNT'][0],
            number_of_active_validators
        ] if x is not None
    )
```

Reply:
Nice! @marthendalnunes an alternative that is shorter, not sure if less intuitive:

```python
number_of_awake_validators: int = min(
  system_parameters.parameters['MAX_VALIDATOR_COUNT'][0] or float('inf'),
  number_of_active_validators
)
```